.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,79 @@
+name: Bug report
+description: File a bug report
+labels: ["bug"]
+
+body:
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce
+      description: Steps to reproduce the bug.
+      value: |
+        1.
+        2.
+        3.
+        4.
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected behaviour
+      description: What should have happened.
+      placeholder: "Example: The specified packages should have been installed."
+    validations:
+      required: true
+
+  - type: textarea
+    id: current-behaviour
+    attributes:
+      label: Current behaviour
+      description: What actually happened. If you were given an error, please provide it here.
+      placeholder: "Example: I received the following error: ..."
+    validations:
+      required: true
+
+  - type: textarea
+    id: version
+    attributes:
+      label: Severn version
+      #description: The version of Severn you're using. Obtainable though `python -m severn`.
```

Comment:
This command will work for this. It just outputs something very simple for now.

Reply:
I didn't even realize you put that in there XD